frame/clearing-house/src/math.rs

```diff
@@ -435,17 +438,17 @@ fn div_rem_with_acc(mut a: u128, mut b: u128, mut acc: u128) -> Result<(u128, u1
 					.expect(
 						"Both `bq` and `acc_num` are stripped. \
 							 If `acc_num` is single-limbed, the previous branch would handle it. \
-							 If `bq` isn't at least one limb bigger than `acc_num`, then `bq` is \
-							 at least one limb smaller than `aa`. But then ",
+							 If `bq` isn't at least one limb bigger than `acc_num`, then ...",
 					)
 					.0
 			}
 			.try_into()
-			.expect("multiplication by quotient is less or equal than original value; qed");
+			.expect("quotient times divisor is less or equal than original value; qed");
+
 			// Then we compute the residual
 			let r = a
 				.checked_sub(a_)
-				.expect("remainder of `a / _` is always less than or equal to a; qed");
+				.expect("remainder of division is always less than or equal to the dividend; qed");
```

Comment:
I'd recommend to add tests ensuring these `.expect` calls never panic either.

Reply:
I'll consider this together with the `kani` integration